setup.py

```diff
@@ -15,7 +15,7 @@
 """Install script for setuptools."""
 
 import os
-from setuptools import find_namespace_packages
+from setuptools import find_packages
```

Comment:
Is this change necessary? I'd rather keep find_namespace_packages if possible.

Reply:
yes, otherwise you need multiple `py.typed` throughout the source tree